Swat/SwatForm.php

```diff
@@ -1109,25 +1003,21 @@ protected function serializeHiddenField($value)
         $value = str_replace('\\', '\\\\', $value);
         $value = str_replace("\x00", '\x00', $value);
         $value = str_replace("\x0a", '\x0a', $value);
-        $value = str_replace("\x0d", '\x0d', $value);
 
-        return $value;
+        return str_replace("\x0d", '\x0d', $value);
```

Comment:
I know PHPStorm says this is simpler, but maintaining the $value= pattern is probably easier to read.

Reply:
It's actually part of the `return_assignment` rule in php-cs-fixer.  I'd need to disable that, although in many cases I feel it is simpler.  I wonder if (in this case) we should just use arrays for the str_replace call, eg.:

```php
        return str_replace(
            ['\\',   "\x00", "\x0a", "\x0d"],
            ['\\\\', '\x00', '\x0a', '\x0d'],
            $value
        );
```